src/box/box.cc

```diff
@@ -779,10 +779,11 @@ box_check_audit(void)
 static void
 box_check_flightrec(void)
 {
-	if (flightrec_check_cfg(
-			cfg_geti64("flightrec_logs_size"),
+	if (flightrec_check_cfg(cfg_geti64("flightrec_logs_size"),
```

Comment:
Nit: useless change. Reverted in a next patch.

Reply:
Fixed.